### PR TITLE
edited orderDao

### PR DESCRIPTION
**Pull request by YiyingLin**

### CODE REVIEW

File: src/main/java/com/feedr/dao/OrderDAO.java [hunk 1]
```diff
@@ -18,15 +18,14 @@ public OrderDAO(DatabaseConnector connector) {
...
+                            Double deliver_tip, Date deadline,
                             String location) throws SQLException{
-        String orderTime = order_time.toString();
         String deadLine = deadline.toString();
         connector.executeQuery(
                 String.format("INSERT INTO order_info (receiver_name,restaurant_name," +
                         "order_cost,deliver_tip,order_time,deadline,delivery_location)" +
-                        " VALUES (%s,%s,%f,%f,%s,%s,%s);", receiName,restName,order_cost,
-                        deliver_tip,orderTime,deadLine,location)
+                        " VALUES (%s,%s,%f,%f,CURRENT_TIMESTAMP,%s,%s);", receiName,restName,order_cost,
```
Comment (omzmarlon) [inline src/main/java/com/feedr/dao/OrderDAO.java]: I'm not sure if this is the correct way to put deadline into the table.
When we insert a datetime into sql, the string has to be in certain format, 
[](https://stackoverflow.com/questions/8551940/mysql-insert-datetime-into-other-datetime-field)
are your sure Java's Date.toString is in this format?

Comment (YiyingLin) [reply src/main/java/com/feedr/dao/OrderDAO.java]: I think it is not the right way, but I pulled it from master and didnt change it.

Comment (omzmarlon) [reply src/main/java/com/feedr/dao/OrderDAO.java]: https://stackoverflow.com/questions/8551940/mysql-insert-datetime-into-other-datetime-field

**Comment by omzmarlon** [comment]
We can fix this later when we write order populations scrips

**Pull request merged**